science/2022-11-23-science-blog-6/index.md

```diff
@@ -4,7 +4,7 @@ page-description: "SB06: What does a red warning mean when incidence is high?"
 page-name: science-blog-6
 page-name_de: science-blog-6
 author: CWA-Team
-date: "2023-02-06"
+date: "2023-02-08"
 update: "last updated November 30, 2022 (German version)"
```

Comment:
There was no reaction to my previous comment about the unclear reference to the German version.

I suggest changing the `update:` parameter to
"from German version (last updated November 30, 2022)"
which makes the final display more understandable when published i.e.

"CWA-Team, on February 8, 2023, from German version (last updated November 30, 2022)"

Reply:
@MikeMcC399 I will forward this.

Reply:
Good morning @MikeMcC399, your suggestion has been accepted. Thanks!

Reply:
Good morning @dsarkar 

> your suggestion has been accepted.

That's good!

For consistency, the other English language science articles dates should also be modified.

Reply:
sure!

Reply:
@MikeMcC399 your suggestion has been implemented in https://github.com/corona-warn-app/cwa-website/pull/3363/commits/880c749b83a36aa3e8815fbb88a153aa158317f0 & a PR for all other science blogs has been made in https://github.com/corona-warn-app/cwa-website/pull/3370